backend/YouTubeV2.Application/DTO/SearchDTOS/SearchResultsDto.cs

```diff
@@ -0,0 +1,6 @@
+﻿using YouTubeV2.Application.DTO.UserDTOS;
+
+namespace YouTubeV2.Application.DTO.SearchDTOS
+{
+    public record SearchResultsDto(IReadOnlyList<UserDto> users);
```

Comment:
since we already have videos implemented we should also return them in serch

Reply:
will do it in separate pr once this is merged